tests/test_cudnn_decode.py

```diff
@@ -102,9 +102,10 @@ def test_cudnn_decode(
         v_cache,
         scale,
         workspace_buffer,
-        actual_seq_lens_kv,
-        block_tables,
-        use_cuda_graph=use_cuda_graph,
+        max_sequence_kv=s_kv,
+        actual_seq_lens_kv=actual_seq_lens_kv,
+        block_tables=block_tables,
+        is_cuda_graph_compatible=is_cuda_graph_compatible,
     )
 
     torch.cuda.synchronize()
```

Comment:
As discussed before we should remove these synchronization for unittests like in prefill kernels.